src/coreclr/jit/valuenum.h

```diff
@@ -690,7 +690,7 @@ class ValueNumStore
 
     // Returns the value number for AllBitsSet of the given "typ".
     // It has an unreached() for a "typ" that has no all bits set value, such as TYP_VOID.
-    ValueNum VNAllBitsForType(var_types typ);
+    ValueNum VNAllBitsForType(var_types typ, unsigned elementCount);
```

Comment:
Document `elementCount`?

Reply:
👍 

I added a description for it matching the existing format since none of the other methods in here are following the more "standard" method documentation format.

It might be desirable to normalize that later, but I went for consistency now.